cmd/srcd/cmd/table.go

```diff
@@ -0,0 +1,53 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"text/tabwriter"
+)
+
+type Table struct {
```

Comment:
We have too many golint warnings about missing docs for exported types and functions.
Let's start to document them please.

Reply:
Added [here](https://github.com/src-d/engine/pull/409/commits/7e03da468499a68ae348b8636b3dec0c44c2e09a).